cmd/test_test.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright (C) 2018 Expedia Group.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"testing"
+	"github.com/stretchr/testify/require"
+	"github.com/spf13/cobra"
+	"bytes"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTestCommand_ShouldExecuteStepAndReturnOutputForJsonInput(t *testing.T) {
```

Comment:
Obviously these test the outline behaviour, but the absence of tests for the details is exactly why I created the trello ticket … 😔